CDP4IME/ShellViewModel.cs

```diff
@@ -387,7 +388,9 @@ private void ExecuteOpenUriManagerRequest()
         /// </summary>
         private void ExecuteOpenPluginManagerRequest()
         {
-            var pluginManager = new PluginManagerViewModel();
+            var appSettingsService = (IAppSettingsService<ImeAppSettings>)ServiceLocator.Current.GetInstance(typeof(IAppSettingsService<ImeAppSettings>));
```

Comment:
I think you can use 
```
ServiceLocator.Current.GetInstance<IAppSettingsService<ImeAppSettings>>()
```
Then you won't need the type cast.

Reply:
The casting is needed is needed since the variable is passed to the view model constructor.

Reply:
I understand that, but when you use the ```GetInstance<T>``` overload (with generic type), you won't need it.
appSettingsService is then already of type ```IAppSettingsService<ImeAppSettings>```, so it doesn't need a cast, and code is shorter, thus better readable.

```
var appSettingsService = ServiceLocator.Current.GetInstance<IAppSettingsService<ImeAppSettings>>();
```
Does exactly the same as 

```
var appSettingsService = (IAppSettingsService<ImeAppSettings>)ServiceLocator.Current.GetInstance(typeof(IAppSettingsService<ImeAppSettings>));
```

Reply:
![image](https://user-images.githubusercontent.com/58699225/75871198-7b54e080-5e0c-11ea-9bcb-230b54525b26.png)

Reply:
Please notice the difference between
``` ServiceLocator.Current.GetInstance<IAppSettingsService<ImeAppSettings>>();```
and 
``` ServiceLocator.Current.GetInstance(typeof(IAppSettingsService<ImeAppSettings>));```

I suggest the first and you're using the second.

Reply:
Cosmetic change applied